### DISCUSSION TITLE
Hiểu ý tưởng giới hạn trong toán học (phần 2) | Math2IT

### DISCUSSION THREAD

**Discussion by giscus** [math2it/math2it-comments#1] [Comments]
# Hiểu ý tưởng giới hạn trong toán học (phần 2) | Math2IT

Phần hai tiếp tục giải đáp những thắc mắc còn dang dở ở phần 1 và nói thêm về sự liên quan giữa ý tưởng giới hạn và các định nghĩa trong SGK chúng ta được học cũng như các ứng dụng thực tế của nó.

http://localhost:3003/bai-viet/hieu-y-tuong-gioi-han-trong-toan-hoc-phan-2/

**Comment by cutamnotcamtu** [comment]
"Giới hạn cho ta niềm tin vào một dự đoán. Cho đến khi bạn không thể chứng minh là tôi sai thì điều tôi nói vẫn đang hữu dụng!" Còn những giải thích thuyết phục trong bài viết đã thắp lên trong em - đứa học sinh cấp 3 đầy non nớt trước biển tri thức một niềm tin mạnh mẽ đặt vào Toán, và vào chính bản thân mình.
